src/edu/uwm/cs/util/MyHashSet.scala

```diff
@@ -1,8 +0,0 @@
-package edu.uwm.cs.util
-
-import scala.collection.mutable.HashSet
-
-class MyHashSet[T] extends HashSet[T] {
```

Comment:
Getting rid of this hand-override is a good idea.  I'm happy that HashSet provides what you need and you found it.